mvds/README.md

```diff
@@ -71,7 +72,7 @@ Each payload contains the following fields:
 **Message Identifiers:** Each `message` has a message identifier calculated by hashing the `group_id`, `timestamp` and `body` fields as follows:
 
 ```
-HASH("MESSAGE_ID", group_id, timestamp, body);
+HASH("MESSAGE_ID", header, group_id, timestamp, body);
```

Comment:
Should this definitely be part of the hash?

Reply:
@oskarth probably worth discussing more in-depth. My current thinking is yes, if I resend the same message but want it to be ackd, should that end the same ID or a different one?